kornia/augmentation/functional.py

```diff
@@ -525,3 +526,139 @@ def _apply_crop(input: torch.Tensor, params: Dict[str, torch.Tensor], return_tra
         params['dst'].to(device).to(dtype),
         return_transform=return_transform
     )
+
+
+def apply_erase_rectangles(input: torch.Tensor, params: Dict[str, torch.Tensor],
+                           return_transform: bool = False) -> UnionType:
+    r"""
+    Generate a {0, 1} mask with drawed rectangle having parameters defined by params
+    and size by input.size()
+
+    Args:
+        input (torch.Tensor): input image.
+        params Dict[str, torch.Tensor]:
+            params['widths'] must be widths tensor
+            params['heights'] must be heights tensor
+            params['xs'] must be x positions tensor
+            params['ys'] must be y positions tensor
+            params['values'] is the value to fill in
+    """
+    if not (params['widths'].size() == params['heights'].size() == params['xs'].size() == params['ys'].size()):
+        raise TypeError(
+            f"''rectangle params components must have same shape"
+        )
+
+    mask = torch.zeros(input.size()).type_as(input)
+    values = torch.zeros(input.size()).type_as(input)
+
+    widths = params['widths']
+    heights = params['heights']
+    xs = params['xs']
+    ys = params['ys']
+    vs = params['values']
+    for i_elem in range(input.size()[0]):
+        h = widths[i_elem].item()
+        w = heights[i_elem].item()
+        y = ys[i_elem].item()
+        x = xs[i_elem].item()
+        v = vs[i_elem].item()
+        mask[i_elem, :, int(y):int(y + w), int(x):int(x + h)] = 1.
+        values[i_elem, :, int(y):int(y + w), int(x):int(x + h)] = v
+    if return_transform:
+        raise NotImplementedError
+    return torch.where(mask == 1., values, input)
+
+
+def apply_adjust_brightness(input: torch.Tensor, params: Dict[str, torch.Tensor],
+                            return_transform: bool = False):
+    """ Wrapper for adjust_brightness for Torchvision-like param settings.
+
+    Args:
+        input (torch.Tensor): Image/Input to be adjusted in the shape of (*, N).
+        brightness_factor (Union[float, torch.Tensor]): Brightness adjust factor per element
+          in the batch. 0 gives a black image, 1 does not modify the input image and 2 gives a
+          white image, while any other number modify the brightness.
+
+    Returns:
+        torch.Tensor: Adjusted image.
+    """
+    if return_transform:
+        raise NotImplementedError
+    return adjust_brightness(input, params['brightness_factor'].type_as(input) - 1)
```

Comment:
why -1 ?

Reply:
Here my logic is, in Kornia, augmentation module is mimicking the behaviour of Torchvision while all other modules keep the original OpenCV style.

The original implementation will output black image if -1, original if 0, white image if 1. While in Torchvision, it output black image if 0, original if 1, brightness enhancement if greater than 1. (It will never output a white image BTW).

I am making this closer to the Torchvision style for all functional and Module-like APIs.